js/views/settingsVw.js

```diff
@@ -91,6 +114,15 @@ module.exports = Backbone.View.extend({
     this.moderatorCount = 0;
 
     this.fetchModel();
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
+      var char = String.fromCharCode(e.which).toLowerCase(),
+          ctrl = (e.ctrlKey || e.metaKey) && !e.altKey;
+
+      if (ctrl && char == config.keyShortcuts.save) {
+        window.obEventBus.trigger('saveCurrentForm', self._state);
```

Comment:
From what I'm seeing, it's not in one place. In this PR, both the settings and userPage views are catching the ctrl-s and each view is triggering a global saveCurrentForm event, which each view then handling their own event.

If we are going to have each view handle this in this way, we just need to get rid of the events, since a view doesn't need to fire an event to itself. It can execute the code directly in the keydown handler -or- call a function.

If we do want to centralize it, then it should probably go back to how Mario had it before, where the key event was captured in main.js and it fires a global event that each view could handle.